### DISCUSSION TITLE
Provide pre-compiled binary release?

### DISCUSSION THREAD

**Discussion by aleung** [hakoerber/git-repo-manager#55] [General]
I was looking for a git worktree CLI helper and found this tool. After reading the document, it looks like it fits my needs and I wanted to give it a try. But I have never developed rust before, it took me almost an hour to setup the toolchain and build it. Downloading everything took a long time, and I probably won't need the build environment after that. If you could provide an executable binary for common OS (e.g. Linux) in the GitHub release for download, it would make it easier for new users.

Thank you very much for making such a good tool.

**Comment by hakoerber** [comment]
Hey,

first, thanks for the feedback!

I completely understand where you are coming from. For me, the rust toolchain is installed anyway, but I see that it can be a real struggle when you have to install it from scratch. Just to make sure: The best way to install the toolchain is via https://rustup.rs/.

About building binaries automatically: I really like the idea. Unfortunately, I currently do not have the bandwidth & knowledge to set this up. I don't want to fight yet another CI tool. If someone steps up and implements the CI config, I am more than happy to merge that.

I opened #59 to track this, as I definitely see the need.

**Comment by hakoerber** [comment]
Hey, I used [`cargo-dist`](https://opensource.axo.dev/cargo-dist/) to create prebuilt, statically linked binaries. See https://github.com/hakoerber/git-repo-manager/releases/tag/v0.7.21

They can either be downloaded manually, or via `cargo-binstall`:

```
cargo binstall git-repo-manager
```
